### PR TITLE
add wandb

### PR DESCRIPTION
**Pull request by nikita-0209**

### CODE REVIEW

**Comment by nikita-0209** [comment]
[wandb log](https://wandb.ai/nikita0209/ActiveLearningPipeline/runs/1pcfdu2g?workspace=user-nikita0209)

File: config/gflownet/gflownet.yaml [hunk 1]
```diff
@@ -80,3 +49,12 @@ policy:
     reload_ckpt: False
   backward: null
   ckpt_period: null
+num_empirical_loss: 200000
+oracle:
+    # Number of samples for oracle metrics
+    n: 500
+# remove eventually
```
Comment (nikita-0209) [inline config/gflownet/gflownet.yaml:56]: note: the following parameters will be removed once we finalize how we'd like to implement these modes

File: gflownet/gflownet.py [hunk 2]
```diff
@@ -210,9 +191,9 @@ def __init__(
         self.ema_alpha = optimizer.ema_alpha
         self.early_stopping = optimizer.early_stopping
         if al_iter >= 0:
-            self.al_iter = "_iter{}".format(al_iter)
+            self.use_context = True
         else:
-            self.al_iter = ""
+            self.use_context = False
```
Comment (alexhernandezgarcia) [inline gflownet/gflownet.py:196]: This can be written in one line: `self.use_context = al_ter >= 0`

File: main.py [hunk 3]
```diff
@@ -15,18 +15,24 @@ def main(config):
     # Log config
     log_config = flatten_config(OmegaConf.to_container(config, resolve=True), sep="/")
     log_config = {"/".join(("config", key)): val for key, val in log_config.items()}
-
+    logger = None
```
Comment (alexhernandezgarcia) [inline main.py:18]: write in else:

**Pull request merged**